plugins/tiddlywiki/katex/wrapper.js

```diff
@@ -70,6 +77,23 @@ KaTeXWidget.prototype.render = function(parent,nextSibling) {
 	// Insert it into the DOM
 	parent.insertBefore(span,nextSibling);
 	this.domNodes.push(span);
+	var tiddlerFrame;
+	span.querySelectorAll('.katex-eqref [data-katex-label]:not([data-katex-eqnum])').forEach(element => {
```

Comment:
For backwards compatibility, the TiddlyWiki core restricts itself to ECMAScript 5.1. Instead of the `forEach` method the core has a `$tw.utils.each()` helper for iteration.